Harvest.Core/Models/AccessCodes.cs

```diff
@@ -7,9 +7,13 @@ public static class AccessCodes
         public const string FieldManagerAccess = "FieldManagerAccess";
         public const string SupervisorAccess = "SupervisorAccess";
         public const string WorkerAccess = "WorkerAccess";
-        /// <summary>
-        /// When using this Authentication, the projectId must be in the parameters.
+        /// <summary>
+        /// When using this Authentication, the projectId must be in the parameters.
         /// </summary>
         public const string PrincipalInvestigator = "PrincipalInvestigator";
+        /// <summary>
+        /// When using this Authentication, the projectId must be in the parameters.
+        /// </summary>
+        public const string PiOnly = "PiOnly";
```

Comment:
use `PrincipalInvestigatorOnly` just to stay consistent with our other `PrincipalInvestigator` access code.